src/nbl/video/SOpenGLContextLocalCache.cpp

```diff
@@ -604,6 +614,13 @@ void SOpenGLContextLocalCache::flushStateGraphics(IOpenGL_FunctionTable* gl, uin
             }
 
             GLuint vao = currentState.vertexInputParams.vaoval.GLname;
+
+            if (currentState.pipeline.graphics.pipeline->getObjectDebugName().size())
+            {
+                const auto& dbgnm = currentState.pipeline.graphics.pipeline->getObjectDebugName();
+                gl->extGlObjectLabel(GL_VERTEX_ARRAY, vao, dbgnm.size(), dbgnm.c_str());
+            }
+
```

Comment:
why is this here? inside of a flush state!?

Reply:
becasue thats where GL object is created

Reply:
decided to set names on GL objects that are shared between nabla objects